ui/pages/confirmations/utils/confirm.ts

```diff
@@ -23,3 +23,12 @@ const SIGNATURE_APPROVAL_TYPES = [
 export const isSignatureApprovalRequest = (
   request: ApprovalRequest<Record<string, Json>>,
 ) => SIGNATURE_APPROVAL_TYPES.includes(request.type as ApprovalType);
+
+const SIGNATURE_TRANSACTION_TYPES = [
+  TransactionType.personalSign,
```

Comment:
In the case of the signature confirmations, is the `type` property actually an `ApprovalType` in which case would it be clearer to define this list using the `ApprovalType` values to decouple it from transactions?

Reply:
I added this method to be used for `currentConfirmation` object, it is not approval request. It comes from MessageManagers / TransactionController - thus I was not sure if its ok to use ApprovalType.

Reply:
I see what you mean.

This makes sense for now but I think it highlights a need to move away from a confirmation directly being alternate controller state objects, and instead define a standardised confirmation type that uses an `ApprovalType` for type and a generic `data` field for anything specific to the type.

This would be very similar to the `ApprovalRequest` so we could even just re-use that.